sysinit/coqloadpath.ml

```diff
@@ -57,6 +58,7 @@ let init_load_path ~coqenv =
       ; coq_path = Libnames.default_root_prefix
       ; implicit = false
       ; recursive = false
+      ; installed = true
```

Comment:
not sure if this one should be installed (this is the implicit -Q . "" which doesn't recurse in subdirs)

Reply:
It'd dare to say this needs to be `installed = false` indeed.

Maybe time to start thinking of removing this, and actually forbidding to bind to an empty path except for the (internal) case of user-contrib

Reply:
I agree it is not installed. I also agree we could consider removing it (but not in the current PR).

Reply:
changed to installed = false